packages/renderer/src/lib/image/ImagesList.svelte

```diff
@@ -214,6 +169,50 @@ function computeInterval(): number {
   // every day
   return 60 * 60 * 24 * SECOND;
 }
+
+let selectedItemsNumber: number;
+let table: Table;
+
+const columns: Column[] = [
+  new Column('Status', 'center', '70px'),
+  new Column('Name', 'left', '3fr'),
+  new Column('Age', 'left', '1fr'),
+  new Column('Size', 'right', '1fr'),
+];
+
+const tableHelper = new (class extends TableHelper {
```

Comment:
As early feedback I would say I'm not finding convenient this kind of declaration from consumer POV

Reply:
I don't love this syntax either. I had an interface before, but I was worried that adding capability in the future (like the previous comparator function) meant that all consumers would be required to change even if they don't care about the functionality yet.

We could stop it at the current capability and extend in other ways. Or maybe switch it to be more like Column where you create an instance and customize via functions? I will keep trying different ways.

Reply:
Pushed another commit to change TableHelper to be more like Column definition. Much simpler for clients to create, please take a look.